docs/reference/analysis/common-uses.asciidoc

```diff
@@ -0,0 +1,98 @@
+== Common uses for analysis
+++++
+<titleabbrev>Common uses</titleabbrev>
+++++
+
+Analyzing full text often means analyzing natural, human language. While each
+language is different, this usually means solving a common set of problems for
+search, such as how to reduce words to their root form.
+
+This section describes how {es} is commonly used to address these problems and
+our recommendations.
+
+[TIP]
+====
+If you'd like to get started quickly, we recommend using {es}'s built-in
+<<analysis-standard-analyzer,`standard`>> analyzer or 
+<<analysis-lang-analyzer,language analyzers>>. These analyzers provide good
+out-of-the-box support for most use cases.
+====
+
+[float]
+[[stemming]]
+=== Stemming
+
+Most natural languages are inflected, meaning words can change
+their form to express differences, such as the following:
+
+* _Number_:      fox, foxes
+* _Tense_:       pay, paid, paying
+* _Person_:      hear, hears
+
+_Stemming_ attempts to remove theses differences and reduce each token to its
+root form. For example, `foxes` can be stemmed to the root `fox`. This often
+aids search retrieval by consolidating near-identical terms.
+
+[float]
+[[stemming-token-filters]]
+==== Stemming token filters
+
+In {es}, stemming is handled by a <<analysis-tokenfilters,token filter>> in
+the <<analyzer-anatomy,analyzer>>.
+
+If you're unsure of which token filter to use for stemming, we recommend
+starting out with the <<analysis-stemmer-tokenfilter,`stemmer`>> filter, which
+supports multiple languages and stemmer types. The
+<<analysis-stemmer-tokenfilter,`stemmer` filter documentation>> page highlights
+the recommended stemmer for each language in bold, based on a reasonable
+compromise between performance and quality.
+
```

Comment:
I'd pare this down to just the recommendation. I'd avoid using "xyz page" or "documentation page" in xrefs--the topic title should be sufficient.

Reply:
I've trimmed this down slightly, but I feel like the introductory phrase is needed here. Otherwise, it could come off as a discouragement of using other stemmers.

I typically agree re: "xyz page" language, but I believe this is a good exception. I'm referring to formatting on the actual page here, not the `stemmer` filter itself. To my knowledge, the actual filter doesn't surface recommended stemmers.